test/test-sock-accept.c

```diff
@@ -0,0 +1,149 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include "uv.h"
+#include "uvwasi.h"
+#include "wasi_serdes.h"
+
+#define PREOPEN_SOCK 3
+#define INVALID_SOCK 42
+
+#define CONNECT_ADDRESS "127.0.0.1"
+#define TEST_PORT_1 10500
```

Comment:
Some of the same questions/comments here as the other test:

- Use static data more.
- Use snake_case.
- Is it possible to use port 0 for the test.
- What about Windows?